WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/PostingActivityMapper.kt

```diff
@@ -55,7 +57,13 @@ class PostingActivityMapper
                                     Calendar.SHORT,
                                     localeManagerWrapper.getLocale()
                             ),
-                            boxes
+                            boxes,
```

Comment:
I needed to create this content description for the Block because the label is the short version of the month so "Dec" is being read as "Deck" so this content description allows me to pass the long version of the month.